packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -122,7 +122,9 @@ ${Object.keys(authorsMap)
       ...author,
       key: author.key ?? null,
       page: author.page ?? null,
-      imageURL: normalizeImageUrl({imageURL: author.imageURL, baseUrl}),
+      imageURL: author.key
+        ? author.imageURL
+        : normalizeImageUrl({imageURL: author.imageURL, baseUrl}),
```

Comment:
The suggested comment is useful to keep and helps understand what's the purpose of this condition

Reply:
We could eventually make things explicit with code:


```js
function normalizeAuthorUrl({author,baseUrl}) {
  if (author.key) {
    if (!author.imageURL && author.imageURL.startsWith(baseUrl)) {
      throw new Error("Docusaurus bug: global authors image ${author.imageURL} should start with the expected baseUrl=${baseUrl}");
    } 
    return author.imageURL;
  }
  return normalizeImageUrl({imageURL: author.imageURL, baseUrl});
}
```

This ensures that the unit tests are always consistent and that the authorsMap images provided always start with baseUrl. This prevents writing inconsistent tests like we currently have